src/main/java/nextstep/courses/domain/session/SessionUsers.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.courses.domain.session;
+
+import nextstep.courses.ExceedMaxAttendanceCountException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SessionUsers {
+
+    private List<NsUser> users = new ArrayList<>();
+    private boolean isFree;
+    private Integer maxAttendance;
+
+    public SessionUsers(boolean isFree, Integer maxAttendance) {
+        this.isFree = isFree;
+        if (isFree) {
+            this.maxAttendance = null;
+            return;
+        }
+        this.maxAttendance = maxAttendance;
+    }
+
+    public void addUser(NsUser nsUser) {
+        if (!isFreeSession() && users.size() >= maxAttendance) {
```

Comment:
```suggestion
        if (!isFreeSession() && users.moreThanSize(maxAttendance)) {
```

값을 꺼내기보다는 비교하라고 시켜볼수 있겠네요!